package-lock.json

```diff
@@ -347,12 +347,30 @@
       }
     },
     "node_modules/@bcrs-shared-components/certify": {
-      "version": "2.1.5",
-      "resolved": "https://registry.npmjs.org/@bcrs-shared-components/certify/-/certify-2.1.5.tgz",
-      "integrity": "sha512-+/6MRAHt1xRTcIauIg6lWDII9sLovl0rr2zAsV3+8zvm9+MVujNqC7mlG0CHI2Czl/tqT+CxpAhm/7m6pOk0+g==",
+      "version": "2.1.55",
+      "resolved": "https://registry.npmjs.org/@bcrs-shared-components/certify/-/certify-2.1.55.tgz",
+      "integrity": "sha512-wcArqfqEtHNQrumVxvmeFwVRkypJgDb4scamGTSFJxvxHJGgCtNKA7xe+i46VQRGnmIpx41T4h2atXLZuQo7yw==",
       "dependencies": {
-        "@bcrs-shared-components/interfaces": "^1.0.48",
-        "vue-property-decorator": "^9.1.2"
+        "@bcrs-shared-components/interfaces": "^1.1.20",
+        "vue": "^2.7.14"
+      }
+    },
+    "node_modules/@bcrs-shared-components/certify/node_modules/@bcrs-shared-components/enums": {
+      "version": "1.1.15",
+      "resolved": "https://registry.npmjs.org/@bcrs-shared-components/enums/-/enums-1.1.15.tgz",
+      "integrity": "sha512-K86ctY66OPPlil+fRTiCeHxNjhSk5PwSn+hNnfdLNsauhzYrWC4MNHuQHcbilOxje6l56JTZtm446TlbPzPK2w==",
+      "dependencies": {
+        "@bcrs-shared-components/corp-type-module": "^1.0.16"
+      }
+    },
+    "node_modules/@bcrs-shared-components/certify/node_modules/@bcrs-shared-components/interfaces": {
+      "version": "1.1.20",
+      "resolved": "https://registry.npmjs.org/@bcrs-shared-components/interfaces/-/interfaces-1.1.20.tgz",
+      "integrity": "sha512-KzxAHYyLANaapMXLuVISFnuNsq2vT56TzK79z38NC3UsAXYalKZDXP5I0zGyKZ8FJIIZi0ZqJLewyHB2ef1dJQ==",
+      "dependencies": {
+        "@bcrs-shared-components/corp-type-module": "^1.0.16",
+        "@bcrs-shared-components/enums": "^1.1.15",
+        "vue": "^2.7.14"
       }
```

Comment:
I ran npm install @bcrs-shared-components/certify, it added those 2 dependencies, is it correct?

Reply:
Yes!

This is because the Certify component depends on Vue and Interfaces:
https://github.com/bcgov/bcrs-shared-components/blob/main/src/components/certify/package.json

And Interfaces depends on Corp Type Module and Enums:
https://github.com/bcgov/bcrs-shared-components/blob/main/src/interfaces/package.json

It's listing Vue as "2.7.14 or greater". Hopefully that \resolves to 2.7.16 (same as Edit UI) instead of importing 2 separate versions of Vue.

Reply:
Anyway, I think this is OK.